src/main/java/io/vertx/core/eventbus/impl/clustered/ClusteredEventBus.java

```diff
@@ -174,6 +174,10 @@ public MessageImpl createMessage(boolean send, String address, MultiMap headers,
     } else if (sendContext.options.isLocalOnly()) {
       super.sendOrPub(sendContext);
     } else {
+      if (isAddressLocal(sendContext.message.address())) {
```

Comment:
If address is also registered locally, then also deliver it using the local EventBus implementation.